app/utils/purecloud-environment.js

```diff
@@ -1,4 +1,4 @@
-const ENV_REG_EXP = /(inin[dts]ca|mypurecloud.*|localhost).*/i;
+const ENV_REG_EXP = /(inin[dts]ca|*.mypurecloud.*|localhost).*/i;
```

Comment:
This regex wasn't correct before, but was working for the wrong reasons. `*.mypurecloud` isn't valid in the new one (`*` is a modifier on the previous character, which is a pipe in this case, which isn't valid). Try this: `((?:inin[dt]ca|localhost|.*\.mypurecloud)\..*)`. You can test regexes using https://www.regexpal.com/.